### ISSUE TITLE
Usage on aarch64 Rasperry Pi

### ISSUE THREAD

**Issue by Tim1605** [BYU-PRISM/GEKKO#139]
Hello,
I would like to use gekko locally on a rasperry pi aarch64.
But unfortunately I get the following error message: 
Exception Type: OSError
Exception Value:	
[Errno 8] Exec format error: '/home/pi/.local/lib/python3.9/site-packages/gekko/bin/apm'

I guess the error is in the gekko.py in line 2089, because there is only searched for the arm architecture for a rasperry pi. I tried to solve the problem by adding the line to:
if os.uname()[4].startswith("arm") or os.uname()[4].startswith("aar"):
changed. However, this causes the following error message to appear:
FileNotFoundError: [Errno 2] No such file or directory: '/home/pi/.local/lib/python3.9/site-packages/gekko/bin/apm_arm'

Would it be possible to enable the local solution on an aarch64?

Thanks a lot

**Comment by APMonitor** [comment]
It looks like the error is that it can't find the apm_arm executable. Could you verify that it is located at /home/pi/.local/lib/python3.9/site-packages/gekko/bin/apm_arm We may need to update the directory location. The executable should work on the arm architecture for the raspberry pi.

**Comment by Tim1605** [comment]
The path /home/pi/.local/lib/python3.9/site-packages/gekko/bin/ contains the following files:

![image](https://user-images.githubusercontent.com/99022473/152770131-fc36dd32-caf8-4123-a52e-ffaefebd961d.png)

**Comment by APMonitor** [comment]
It looks like the executable is there and it is the correct path. Are there any other errors besides the one that you reported?

```
FileNotFoundError: [Errno 2] No such file or directory: '/home/pi/.local/lib/python3.9/site-packages/gekko/bin/apm_arm'
```

**Comment by Tim1605** [comment]
No, after adding the line there is only this error

**Comment by APMonitor** [comment]
What is the Raspberry Pi version that you are using?

**Comment by Tim1605** [comment]
I'm using a Raspberry Pi 3B+

**Comment by Tim1605** [comment]
Hello, is there a solution in sight?

**Comment by APMonitor** [comment]
It looks like the binary for apm_arm may be compatible with the device as shown at https://unix.stackexchange.com/questions/461179/what-is-the-difference-between-different-implemetation-of-arm64-aarch64-for-linu I don't have a Raspberry Pi 3+ for testing currently but I'll let you know as soon as there are some developments.

**Comment by APMonitor** [comment]
A new version of the `apm_arm` file is released with `gekko` `v1.0.4`. Could you test it and see if it works for you? It is a 32-bit executable, compiled on a Raspberry Pi for compatibility for Raspberry Pi 2, 3, and 4.

**Comment by APMonitor** [comment]
A new `aarch64` binary is currently available on GitHub and will be included with `gekko` v1.0.7 soon.
